test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -186,21 +186,21 @@ TEST_F(AmetsuchiTest, SampleTest) {
   iroha::Amount amount;
 
   // Block 1
-  auto block1 =
-      TestBlockBuilder()
-          .transactions(std::vector<shared_model::proto::Transaction>(
-              {TestTransactionBuilder()
-                   .creatorAccountId("admin1")
-                   .createRole(
-                       "user",
-                       std::set<std::string>{
-                           iroha::model::can_add_peer, iroha::model::can_create_asset, iroha::model::can_get_my_account})
-                   .createDomain(domain, "user")
-                   .createAccount(user1name, domain, fake_pubkey)
-                   .build()}))
-          .height(1)
-          .prevHash(fake_hash)
-          .build();
+  auto block1 = TestBlockBuilder()
+                    .transactions(std::vector<shared_model::proto::Transaction>(
+                        {TestTransactionBuilder()
+                             .creatorAccountId("admin1")
+                             .createRole("user",
+                                         std::set<std::string>{
```

Comment:
There is need to be some specific alias type, something like `PermissionSetType`. I suggest to add todo for it and go ahead.

Reply:
Find a task, that related to the reworking of permissions and add it here.